src/components/NavButtons.tsx

```diff
@@ -4,20 +4,24 @@ import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
 import { PageOrder } from '../types/globalTypes';
 export interface NavProps {
   page: number;
+  next: boolean;
+  prev: boolean;
```

Comment:
Instead of making these mandatory, you can make them optional and either set a default value or make them appear unless next/prev are set to false.